### ISSUE TITLE
Publish Angular Dashboard App to GitHub

### ISSUE THREAD

**Issue by matthabermehl** [ady624/CoRE#33]
Do you have a separate git repo for the angular app you're using for the Dashboard? If so, where is it so that I may fork it? I'd love to toss in some front-end love once in a while, and though I can get all the source it would be nice to have a formal pull request mechanism. Thanks!

**Comment by ady624** [comment]
The only reason it's not on github is because rawgit would do a permanent cache and the code is still being changed. I can post it to github but I won't make it use github yet. Caching issues would become a nightmare... They're a few static js, html and css files, you can find them all with the browser's console. I think I also use a font file.

**Comment by matthabermehl** [comment]
Ok Cool. I'll keep an eye out and just contact you here if I have any contributions (just about to start a new job, so not sure how soon that will be :-)). Thanks for all your work on this project, btw. Just discovered it, and it's exactly the killer app that ST needs.
Has there been any thought given to having a web interface with a friendlier UI to build the various rules?
Thanks,
Matt

**Comment by talz13** [comment]
@matthabermehl Yes, I would love to see a web ui that I could edit pistons from a 20"+ screen, and see everything at once, instead of having to do everything on a (albeit large for a phone) 6" screen.

**Comment by idpaterson** [comment]
@ady624, once CoRE is released if you were to tag each release (e.g. git-flow) would it be possible for `/dashboard` to reference the static files based on the version of CoRE installed? Something like `https://cdn.rawgit.com/ady624/CoRE/1.0.4/dashboard/js/app.js`?

I am not sure whether the web interface was envisioned as living in this repository but if so, tagged releases for both the smart app and the dashboard may be a convenient approach. In addition to opening the dashboard source and deployment to contributors it would allow you to introduce interdependent features without tiptoeing around backwards compatibility for users on older versions of CoRE. In this case, a permanent CDN cache becomes a feature rather than a burden.

**Comment by ady624** [comment]
It is meant to be on the cdn. The code is open source, the browser loads it all, there is no server side code, it's all made up of html, css, and js files. I will move them to github when I get some time, maybe this weekend.
